### PR TITLE
[BPK-1674] Move auto import logic to use-tokens rule

### PR DESCRIPTION
**Pull request by tiagoengel**
This PR extracts the auto import logic to reusable functions and add it as part of the `use-tokens` rule. The plan is to apply the auto import logic in the `use-components` rule too.

### CODE REVIEW

File: src/rules/use-tokens.js [hunk 1]
```diff
@@ -118,7 +128,26 @@ const checkColor = (node, context) => {
       message: `Use the following Backpack token instead: ${
         expectedToken.name
       }`,
-      fix: fixer => fixer.replaceText(value, expectedToken.name),
+      fix: fixer => {
+        const { options: userOptions } = context;
+        const options = _.merge({}, BASE_CONFIG, userOptions[0] || {});
+        const tokensPkg = options.tokensPackage[options.platform];
+
+        const importDefinition = getImportDefinition(
```
Comment (k0nserv) [inline src/rules/use-tokens.js]: nitpic: We don't need to construct this value if we have `!options.autoImport`.

File: changelog.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+### Added
```
Comment (shaundon) [inline changelog.md]: This probably comes under 'removed', I think.

Comment (tiagoengel) [reply changelog.md]: I'll address this on #10

File: src/rules/use-tokens.js (hunk 1 above)
Comment (tiagoengel) [reply src/rules/use-tokens.js]: I'll address this on #10

**Pull request merged**